packages/block-template/webpack-main.config.js

```diff
@@ -20,9 +20,11 @@ module.exports = {
       openAnalyzer: false,
       reportFilename: "webpack-bundle-analyzer-report.html",
     }),
-    new WebpackAssetsManifest(),
+    new WebpackAssetsManifest({
+      output: "manifest.json",
```

Comment:
The default value [was changed](https://github.com/webdeveric/webpack-assets-manifest#new-in-version-5) to `output-manifest.json`. I added a config option for better backwards-compatibility, but I can remove it now or revisit later.